tests/test_browser_find_overlay_portal_regression.py

```
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+"""Regression guards for browser Cmd+F overlay layering in portal mode."""
+
+from __future__ import annotations
+
+import subprocess
+from pathlib import Path
+
+
+def repo_root() -> Path:
+    result = subprocess.run(
+        ["git", "rev-parse", "--show-toplevel"],
```

Ruff (0.15.2)

[error] Starting a process with a partial executable path

(S607)